packages/graphql-codegen/src/generateGraphqlSchemaFiles.test.ts

```diff
@@ -0,0 +1,224 @@
+import { snakeCase } from "change-case";
+import { EntityDbMetadata, makeEntity, PrimitiveField } from "joist-codegen";
+import { plural } from "pluralize";
+import { generateGraphqlSchemaFiles } from "./generateGraphqlSchemaFiles";
+import { Fs } from "./utils";
+
+describe("generateGraphqlSchemaFiles", () => {
```

Comment:
This unit test is turning out to be pretty useful b/c the `.history.json` approach means that just running `npm run codegen` in `integration-tests` can't exercise all of the boundary cases of "this is a not-yet-added field / etc.".